app/src/routes/login/components/continue-as.vue

```diff
@@ -49,6 +49,7 @@ async function fetchUser() {
 
 async function hydrateAndLogin() {
 	await hydrate();
+	await userPromise;
```

Comment:
Why are not calling the fetchUser() function here directly?

Reply:
It's a race condition as the function had already ran. We should not run again as it's unnecessary.